src/components/StudentSelect.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <p>SELECT YOUR SCHOOL:</p>
+  <select v-model="selectedSchool" @change="changeSchool(event)">
+    <option disabled value="">Please select your school</option>
+    <option v-for="school in this.schoolsList" :key="school.id" :value="school.name">
```

Comment:
```suggestion
    <option v-for="school in schoolsList" :key="school.id" :value="school.name">
```
`this` keyword is not required while writing logic in html in vue